pkgs/development/tools/npm-check/default.nix

```
@@ -0,0 +1,29 @@
+{ buildNpmPackage
+, fetchFromGitHub
+, lib
+}:
+
+buildNpmPackage rec {
+  pname = "npm-check";
+  version = "6.0.1";
+
+  src = fetchFromGitHub {
+    owner = "dylang";
+    repo = pname;
+    rev = "v" + (toString version);
```

```suggestion
    rev = "v${version}"
```
We can use string interpolation here.